examples/keyboard/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# all source files
+set(SRC Keyboard.cpp)
+
+# define the keyboard target
+sfml_add_example(keyboard GUI_APP
+                 SOURCES ${SRC}
+                 DEPENDS SFML::Audio SFML::Graphics
+                 RESOURCES_DIR resources)
```

Comment:
I copy-pasted this from other examples, but it looks like it is simply not used anywhere.
This was used for example install rules but examples are not installed anymore since #1576.

Reply:
I removed `RESOURCES_DIR` here.